paddle/fluid/pybind/pybind_boost_headers.h

```diff
@@ -0,0 +1,115 @@
+/* Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
why we need this

Reply:
Because Pybind11 needs custom boost::variant type caster in C++ 11, this could be removed in C++14